stable/minio/templates/deployment.yaml

```diff
@@ -78,12 +78,12 @@ spec:
             - name: MINIO_ACCESS_KEY
               valueFrom:
                 secretKeyRef:
-                  name: {{ template "minio.fullname" . }}
+                  name: {{ if .Values.createSecret }}{{ template "minio.fullname" . }}{{ else }}{{ .Values.existingSecretName }}{{ end }}
```

Comment:
Please drop `createSecret`, rename `existingSecretName` to `existingSecret`, and use it if set. You can find many examples that do it that way. Please follow this style. Also, please create a helper template for the secret name.

Here's an example:
https://github.com/helm/charts/blob/cf7a5644c63d7481528206fe59939ef4ae3a119e/stable/openldap/templates/_helpers.tpl#L35-L40
https://github.com/helm/charts/blob/cf7a5644c63d7481528206fe59939ef4ae3a119e/stable/openldap/templates/deployment.yaml#L43

Reply:
I vaguely recalled having a reason for using 2 variables but I don't know what it was. Either, way updated, only using a single variable now!